dist/files/ovn-kubernetes-master.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=ovn-kubernetes master service
+Requires=atomic-openshift-master-api.service
```

Comment:
Not sure about mentioning openshift here. We do depend upon the api server to be existing here. But assuming a dependency on another unit sounds misplaced to me.
@dcbw Shall we omit this? And assume that it will be handled well by the code only? Anyhow, this should not assume that api server is running locally.

Reply:
Remove the dependency